Makefile

```diff
@@ -38,12 +30,11 @@ dist: all
 	docker save $(IMAGE_NAME) | gzip > $(DISTFILE)
 
 # Build the container image.
+build: export DOCKER_BUILDKIT = 1
 build:
```

Comment:
Nit: I think you can do `DOCKER_BUILDKIT=1 docker build` instead of adding an extra `build`. I'm not a Make expert so you may have good reasons to add the extra `build:`, but nothing other than the `docker` command will use the environment variable so it should be OK not to export it.

Reply:
That makes sense to me. I was just copying what we had in the admin container, but I'll make this change for both.